getting-started.md

```diff
@@ -55,12 +35,11 @@ the `flutter` command.)
 After you clone the Flutter repo, set the PATH so you can
 use our scripts and tools.
 
-```
-$ cd <directory where you cloned the flutter repo>
-$ export PATH=`pwd`/bin:$PATH
-```
+`export PATH=` _directory where you cloned the flutter repo_ `/bin:$PATH`
 
-Run `flutter --version` to ensure the `flutter` command is on your PATH.
+Open a new shell and run `flutter --version` to ensure the `flutter` command is on your PATH.
+The first time you do this will take a few seconds as Flutter will first download the Dart SDK then precompile itself.
+Subsequent runs should be much faster.
```

Comment:
Flutter will only need to do these steps after subsequent `flutter upgrade` commands?